collections/Laud_Misc/MS_Laud_Misc_97.xml

```diff
@@ -37,46 +36,256 @@
                      <idno type="SCN">1213</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>Origen, Homiles on Joshua. Germany (Lorsch), 9th century, first half</head>
                <msContents>
                   <msItem n="1" xml:id="MS_Laud_Misc_97-item1">
                      <author key="person_95155322">Origen</author>
                      <title key="work_3395">Homilies on Joshua</title>
-                     <note>(tr. <persName key="person_77679446" role="trl"> Rufinus </persName>)</note>
+                     <note>(tr. <persName key="person_77679446" role="trl"> Rufinus
+                        </persName>)</note>
+                     <note>Ed. W. A. Baehrens, GCS 30 (1921), pp. 286-463; the quires misbound after fol. 29, as noted below.</note><!-- https://archive.org/details/origeneswerke07orig/page/352/mode/2up -->
+                    <msItem>
+                       <note>Fols. 1-29: quires 1-4 (a-d)</note>
+                       <msItem>
+                        <locus>(fol. 1r-v)</locus>
+                          <rubric>Incipit prologus Hieronimi super librum Iesu Naue ad Chromacium episcopum</rubric>
```

Comment:
Is it worth marking Rufinus as the author of the prologue? The similar entry at <https://medieval.bodleian.ox.ac.uk/catalog/manuscript_10103> does not do so, but also does not catalogue the sections individually.